src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileParser.cs

```diff
@@ -112,19 +108,16 @@ private static void ScavengeCache()
 				// Force the line info, otherwise it will be enabled only when the debugger is attached.
 				var settings = new XamlXmlReaderSettings() { ProvideLineInfo = true };
 
-				(XmlReader document, bool disableCaching) = ApplyIgnorables(file, sourceText, cancellationToken);
+				XmlReader document = ApplyIgnorables(sourceText);
 
-				using (var reader = new XamlXmlReader(document, context, settings))
+				using (var reader = new XamlXmlReader(document, context, settings, IsIncluded))
 				{
 					if (reader.Read())
 					{
 						cancellationToken.ThrowIfCancellationRequested();
 
 						var xamlFileDefinition = Visit(reader, file.Path, targetFilePath);
-						if (!disableCaching)
-						{
-							_cachedFiles[cachedFileKey] = new CachedFile(DateTimeOffset.Now, xamlFileDefinition);
-						}
```

Comment:
This is NOT correct. I need to find a way to restore the `disableCaching` condition, which should be why the CI is currently failing.

I had a wrong assumption that `disableCaching` can be removed.